docs/en/wallet-solution.rst

```diff
@@ -5,99 +5,91 @@
 Wallet Solution
 -------------------
 
-The Wallet Solution is a comprehensive product offered by the Wallet Provider to cater to the needs of Users in managing their digital assets securely. It is issued by the Wallet Provider in the form of a mobile app, it also consists of services and web interfaces for the exchange of data between the Wallet Provider and its Wallet Instances for the requirements of the trust model and in total respect of the User's privacy, in accordance with national and EU legislation.
+The Wallet Solution is a comprehensive product offered by the Wallet Provider to cater to the needs of Users in managing their digital assets securely. It is issued by the Wallet Provider in the form of a mobile app and consists of services and web interfaces for the exchange of data between the Wallet Provider and its Wallet Instances to meet the requirements of the trust model and ensure full respect for the User's privacy, in accordance with national and EU legislation.
```

Comment:
I suggest to remove definitions from here that are already defined in the defined terms section. I suggest to improve the defined term section instead.

Reply:
anyway, I would feel more comfortable in doing this in 0.8.1

we should handle an issue for editorial revision to reduce definition duplication along the sections

Reply:
I would leave the introduction for now, then maybe we can fix it better in the defined terms. It is used to make it clear to the user what we are going to talk about

Reply:
I agree